R/readFEGECA.R

```diff
@@ -0,0 +1,50 @@
+#' Read heating systems sales in Spain from FEGECA
+#'
+#' @param subtype Character, report to read. if NULL, the most recent report is
+#'   used for every data point.
+#' @returns magpie object
+#'
+#' @author Robin Hasse
+#'
+#' @importFrom utils read.csv
+#' @importFrom dplyr %>% mutate filter select across all_of row_number arrange n
+#' @importFrom magclass as.magpie
+#' @importFrom quitte as.quitte
+#' @export
+
+readFEGECA <- function(subtype = NULL) {
+
+  # names of files must be orderable
+  files <- c(`2015` = "2017_2018_2019.csv",
+             `2020` = "2020.csv",
+             `2020` = "2021.csv",
+             `2020` = "2022.csv",
+             `2022` = "2023.csv")
```

Comment:
Without knowing the data, this mapping seems rather counter-intuitive - but probably there is a good reason for these names? (I just want to remark this in the unlikely case that there is actually an error here).

Reply:
You're right - copy-paste error...